publication-commons/src/main/java/no/unit/nva/publication/model/storage/FileDao.java

```diff
@@ -164,12 +164,16 @@ protected User getOwner() {
     @Override
     @JsonProperty(BY_TYPE_AND_IDENTIFIER_INDEX_PARTITION_KEY_NAME)
     public String getByTypeAndIdentifierPartitionKey() {
-        return KEY_PATTERN.formatted(TYPE, getIdentifier());
+        return KEY_PATTERN.formatted(Resource.TYPE, getResourceIdentifier());
```

Comment:
Updating keys in order to be able to fetch Resource and Files by single DynamoDB query.